.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'bug: '
+labels: bug 🐛
+---
+
+<!-- 
+    Before submitting an issue, please consult our [documentation](https://decathlon.design). 
+    Please make sure you are posting an issue pertaining to the Decathlon Design System. 
+
+    Note that before opening an issue, you can chat maintainer of this project on channel
+    #vitamin-ios in [Decathlon Design System's Slack(https://join.slack.com/t/decathlon-design/shared_invite/zt-ou0n9qas-n_oamDSVUIqvLqNO1LETJg).
+
+    If you want to know how to contribute to this project, you can check our CONTRIBUTING file:
+    https://github.com/Decathlon/vitamin-ios/blob/main/CONTRIBUTING.md
+
+    If a section isn't adapted for your request, please remove it to avoid any unnecessary section.
+
+    Thanks!
+-->
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**Steps to reproduce**
+<!--
+    Steps to reproduce the behavior:
+    1. Go to '...'
+    2. Click on '....'
+    3. Scroll down to '....'
+    4. See error
+-->
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Browsers affected**
+
+<!-- Choose your browser(s):
```

Comment:
Should we replace with iOS version ? Or phone model ? or both ?

Reply:
you're right, fixed!